scripts/scr_initialize_custom/scr_initialize_custom.gml

```diff
@@ -1023,11 +1029,91 @@ function scr_initialize_custom() {
 					}
 				}
 			],
-			[role[100,18], {"max":1,"min":1, "role":"Assualt Sergeant"}],		// sergeant
-			["display_name" , "Tactical Squad"]
-		]		
+			[role[100,18], {"max":1,"min":1, "role":$"Assualt {role[100,18]}"}],		// sergeant
+			["display_name" , $"{role[100,10]} {squad_name}"]
+		],
+	    "scout_squad":[
```

Comment:
oooh yeah, I like where this is going. Data-driven is definitely the way to go for CM